pkg/controller/gitopsservice/gitopsservice_controller.go

```diff
@@ -229,6 +231,23 @@ func newDeploymentForCR(cr *pipelinesv1alpha1.GitopsService) *appsv1.Deployment
 						Value: insecureEnvVarValue,
 					},
 				},
+				VolumeMounts: []corev1.VolumeMount{
+					{
+						MountPath: "/etc/gitops/ssl",
+						Name:      "backend-ssl",
+						ReadOnly:  true,
+					},
+				},
+			},
+		},
+		Volumes: []corev1.Volume{
+			{
+				Name: "backend-ssl",
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: "pipelines-app-delivery-backend",
```

Comment:
The secret name should be `cr.Name`.  It should match the value provided in the annotation `"service.beta.openshift.io/serving-cert-secret-name"`